code/script.js

```diff
@@ -1,19 +1,150 @@
 // Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
 
+const userName = prompt(
+  `Hi hungry fellow! What's your name?`
+)
+
+alert(
+  `Hello, ${userName}! Hit the button below whenever you are ready to order!`
+)
+
 // Step 2 - Food choice
-// Your code goes here
+const mealchoice = prompt(
+  `What would you like to order? Please enter the number of your choice.
+  
+  1. Pizza
+  2. Pasta
+  3. Salad
+  `
+)
+ //Pizza
+if (mealchoice === "1") {
```

Comment:
I saw you mentioned this in your README as well, regarding indentation. If you've turned off the default formatting, make sure to have another look through your file and indent everything properly, e.g.
```
if (mealchoice === "1") {
  const pizzachoice = prompt(
    `Thanks, ${userName}! You wish to order Pizza!
      
     What type of Pizza you like to order? Please enter the number of your choice.
    
        1. Capricciosa 
        2. Hawaii
        3. Vesuvius
     `)
  if (pizzachoice === "1") {
    orderedFood = "Capricciosa pizza"
  } else if (pizzachoice === "2") {
    orderedFood = "Hawaii pizza"
  } else if (pizzachoice === "3") {
    orderedFood = "Vesuvius pizza"
  } else {
    alert("Sorry, we don't have that kind of pizza.")
  }
  if (orderedFood) {
    alert(`You've chosen ${orderedFood}`)
  }
} else if....
  ```